File: src/main/java/no/nav/sbl/dialogarena/modiasyforest/rest/domain/adresse/Matrikkeladresse.java
package no.nav.sbl.dialogarena.modiasyforest.rest.domain.adresse;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class Matrikkeladresse extends StrukturertAdresse {
    public String kommunenummer;
    public String gardsnummer;
    public String bruksnummer;
    public String festenummer;
    public String seksjonsnummer;
    public String undernummer;
    public String eiendomsnavn;
    public String postnummer;
}

File: src/main/kotlin/no/nav/syfo/controller/user/domain/adresse/Matrikkeladresse.kt
package no.nav.syfo.controller.user.domain.adresse

class Matrikkeladresse(
    val kommunenummer: String? = null,
    val gardsnummer: String? = null,
    val bruksnummer: String? = null,
    val festenummer: String? = null,
    val seksjonsnummer: String? = null,
    val undernummer: String? = null,
    val eiendomsnavn: String? = null,
    val postnummer: String? = null
) : StrukturertAdresse()
